awx/api/metadata.py

```diff
@@ -23,9 +23,12 @@
 # AWX
 from awx.api.fields import ChoiceNullField
 from awx.main.fields import JSONField, ImplicitRoleField
-from awx.main.models import NotificationTemplate
+from awx.main.models import NotificationTemplate, UnifiedJob, WorkflowApproval, UnifiedJobTemplate, Project, SystemJobTemplate
```

Comment:
Can you revert this part? I don't think we need it anymore after your last commit.

Reply:
Sure, thanks for your reminder!